pkg/tests/tasks/security/authorization/ext_auth_test.go

```diff
@@ -129,7 +211,27 @@ spec:
     - operation:
        paths: ["/headers"]
 `
-
+  ExternalRouteGrpc = `
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: ext-authz
+  namespace: foo
+spec:
+  selector:
+    matchLabels:
+      app: httpbin
+  action: CUSTOM
+  provider:
+    # The provider name must match the extension provider defined in the mesh config.
+    # You can also replace this with sample-ext-authz-http to test the other external authorizer definition.
+    name: sample-ext-authz-grpc
+  rules:
+  # The rules specify when to trigger the external authorizer.
+  - to:
+    - operation:
+       paths: ["/headers"]
```

Comment:
This indentation should cause failure on applying the object. Very interesting that it didn't fail. You can also fix corresponding configuration for `sample-ext-authz-http`.
```suggestion
    - operation:
        paths: ["/headers"]
```

Reply:
You resolved the comment, but it seems to not be done. Could you fix this indentation?